src/NextBuilds.php

```diff
@@ -161,6 +162,20 @@ function (ModelEvent $event) {
                         $revalidateMenu = ($entry->type->handle == "pages");
                         Craft::$app->onAfterRequest(function() use ($entry, $revalidateMenu) {
                             $this->request->buildPagesFromEntry($entry->uri, $revalidateMenu);
+                            if (!is_null($this->settings->getEnableCDNCacheInvalidation()))
+                            {
+                                if (!$this->settings->getEnableCDNCacheInvalidation())
```

Comment:
Reverse your logic here, rather than asset that CDN caching is not enabled and simply logging an error if so, assert that it is set and make the call to the method and leave the else for logging